apps/mobile/src/ui/Analytics/Analytics.tsx

```diff
@@ -235,7 +235,11 @@ const CommonContent = ({
           >
             <Text style={style}>{icon}</Text>
 
-            <Text style={{...a.body_1_lg_regular}}>{strings.ui[key]}</Text>
+            <Text style={{...a.body_1_lg_regular}}>
```

Comment:
ta not a. (im guessing)
```suggestion
            <Text style={[a.body_1_lg_regular]}>
```